diff --git a/ruby/private/rspec.bzl b/ruby/private/rspec.bzl
index 42a9378..9ee4327 100644
--- a/ruby/private/rspec.bzl
+++ b/ruby/private/rspec.bzl
@@ -10,17 +10,14 @@ load(":binary.bzl", "ruby_binary_macro")
 
 def ruby_rspec(
         name,
-        srcs,
         specs,
+        srcs = None,
         deps = None,
         size = "small",
         rspec_args = None,  # This is expected to be a dictionary
         bundle = DEFAULT_BUNDLE_NAME,
         visibility = None,
         **kwargs):
-    if specs == None:
-        specs = []
-
     if srcs == None:
         srcs = []
 
@@ -41,7 +38,7 @@ def ruby_rspec(
         else:
             args_list.append("%s" % (option))
 
-    args_list += specs
+    args_list += ["$(location " + s + ")" for s in specs]
 
     rspec_gems = ["%s:%s" % (bundle, gem) for gem in DEFAULT_RSPEC_GEMS]
 
