diff --git a/repositories.bzl b/repositories.bzl
index cced1d29b..94dbc357f 100644
--- a/repositories.bzl
+++ b/repositories.bzl
@@ -43,7 +43,7 @@ IO_GRPC_GRPC_JAVA_ARTIFACTS = [
     "io.opencensus:opencensus-api:0.24.0",
     "io.opencensus:opencensus-contrib-grpc-metrics:0.24.0",
     "io.perfmark:perfmark-api:0.25.0",
-    "junit:junit:4.12",
+    "junit:junit:4.13.2",
     "org.apache.tomcat:annotations-api:6.0.53",
     "org.codehaus.mojo:animal-sniffer-annotations:1.21",
 ]
